tools/docker/Dockerfile

```diff
@@ -98,6 +98,15 @@ RUN wget -nv https://www.nordicsemi.com/-/media/Software-and-other-downloads/Des
   dpkg -i /tmp/nrf-cli-tools/nRF-Command-Line-Tools_10_12_1_Linux-amd64.deb && \
   rm -rf nRFCommandLineTools10121Linuxamd64.tar.gz /tmp/nrf-cli-tools
 
+ARG COMMANDER_VERSION=1v13p1b1154
+RUN wget -nv https://www.silabs.com/documents/public/software/SimplicityCommander-Linux.zip && \
+  unzip SimplicityCommander-Linux.zip && \
+  mkdir /opt/Commander_linux_x86_64_${COMMANDER_VERSION} && \
+  tar xf SimplicityCommander-Linux/Commander_linux_x86_64_${COMMANDER_VERSION}.tar.bz -C /opt/Commander_linux_x86_64_${COMMANDER_VERSION} && \
```

Comment:
Is there any point in having the commander subdirectory under the newly created directory in /opt? If not, you can add `--strip-components=1` to the tar command.

I think this is still in the phase of the docker building that is running as root, so tar should probably also be called with ` --no-same-owner` so the installed files aren't owned by "toolsadmin".

Reply:
Done